src/serac/physics/utilities/variational_form/detail/timer.hpp

```diff
@@ -0,0 +1,16 @@
+#include <chrono>
+
+class timer {
```

Comment:
Should we be using caliper for this? I think mfem also has a timer class that is pretty similar.

Reply:
https://github.com/LLNL/axom/blob/develop/src/axom/core/utilities/Timer.hpp

so does axom =)

Reply:
Yeah, we should probably use the axom one instead of re-implementing it.